README.md

```diff
@@ -1,5 +1,6 @@
 # Protego
 
+![build-badge](https://img.shields.io/travis/anubhavp28/protego)
```

Comment:
https://img.shields.io/travis/scrapy/protego ?

Reply:
Why it still says "build unknown" when Travis is passing all the tests? Is this even the right way to integrate badges?

Reply:
My guess would be that it only takes the `master` branch into account.